config/project/fields/callToActionBlock--7f2e3eef-0a17-44d0-b7a2-06e593deebf5.yaml

```diff
@@ -0,0 +1,28 @@
+columnSuffix: null
+handle: callToActionBlock
```

Comment:
Should this be called a _block_ as well since it isn’t a part of the page builder toolkit? If yes (as in it is indeed a part of that toolkit as well) then shouldn’t we follow the same handle naming convention?